kivy/config.py

```diff
@@ -805,6 +811,14 @@ def name(self, value):
         elif version == 15:
             Config.setdefault('kivy', 'kivy_clock', 'default')
 
+        elif version == 16:
+            Config.setdefault('kivy', 'default_font', [
```

Comment:
Yup and `literal_eval` comes to the rescue for converting it back 🍡 . It might be included in Config though.